src/SortableTrait.php

```diff
@@ -47,6 +49,8 @@ public static function setNewOrder($ids, int $startOrder = 1, string $primaryKey
         }
 
         $model = new static();
+        
+        Sorting::dispatch($model);
```

Comment:
Use the `event()` function to dispatch the new event.

Reply:
✅